crates/subspace-service/src/lib.rs

```diff
@@ -35,6 +37,7 @@ use sc_consensus_subspace::{
     RewardSigningNotification, SubspaceLink, SubspaceParams,
 };
 use sc_executor::{NativeElseWasmExecutor, NativeExecutionDispatch};
+pub use sc_network::Keypair;
```

Comment:
Why do we need this re-export? I'd prefer to not have this and below re-exports, they have a tendency of being forgotten afterwards. Better add `subspace-networking` to explicit dependencies of `subspace-node`.